src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -188,31 +189,23 @@ const VirtualMachinesList: FC<VirtualMachinesListProps> = forwardRef((props, ref
   const existingSelectedVMs = useExistingSelectedVMs(filteredVMs);
   const isAllVMsSelected = filteredVMs?.length === existingSelectedVMs.length;
 
-  const allVMs = useMemo(
-    () => filterVMsByClusterAndNamespace(vmsSignal.value, namespace, cluster),
+  const hasNoVMs = useMemo(
+    () => isEmpty(filterVMsByClusterAndNamespace(vmsSignal.value, namespace, cluster)),
     [vmsSignal.value, namespace, cluster],
   );
 
-  const hasNoVMs = isEmpty(allVMs);
-
   return (
     /* All of this table and components should be replaced to our own fitted components */
     <>
       <DocumentTitle>{PageTitles.VirtualMachines}</DocumentTitle>
       {!isSearchResultsPage && (
-        <>
-          <PageBreadcrumb>
-            <VirtualMachineBreadcrumb cluster={cluster} namespace={namespace} />
-          </PageBreadcrumb>
-          <PageSection>
-            <VirtualMachineListSummary
-              namespace={namespace}
-              onFilterChange={onFilterChange}
-              vmis={vmis}
-              vms={allVMs}
-            />
-          </PageSection>
-        </>
+        <PageSection>
+          <VirtualMachineListSummary
+            onFilterChange={onFilterChange}
+            vmis={vmis}
+            vms={filteredVMs}
```

Comment:
this means that we'll show summary of only filtered vms ? ALso based on row filters like os and status ?